instructions/4-create-postgres-on-azure.md

```diff
@@ -105,6 +105,7 @@ It should report:
 ```
 <path>/setup/postgresql.jar: Java archive data (JAR)
 ```
+
 ### Create Application Settings

```

Comment:
Does this need a decimal instead of `###` ?

Reply:
Good catch! Yeah will fix that